src/sage/manifolds/vector_bundle.py

```diff
@@ -796,7 +796,7 @@ def local_frame(self, *args, **kwargs):
                 resu._init_from_family(sections)
             except ArithmeticError as err:
                 linked = str(err) in ["non-invertible matrix",
-                                      "input matrix must be nonsingular"]
+                                      "input matrix must be non-singular"]
```

Comment:
```suggestion
                                      "input matrix must be nonsingular"]
```
I prefer nonsingular, and, similarly, nonnegative etc. It is also written without a hyphen on wiki and wolfram.